.github/workflows/cmake.yml

```diff
@@ -209,6 +213,11 @@ jobs:
             packages: wine-stable gcc-mingw-w64
             codecov: ubuntu_gcc_mingw_x86_64
 
+          - name: Ubuntu 16.04 Clang
+            os: ubuntu-16.04
+            compiler: clang-6.0
```

Comment:
I think we can use `ubuntu-latest` so long as we have `package: clang-6.0`, correct?

Reply:
The "Minimum requirements" specifies operating systems, and this is a token effort of
testing that zlib supports clang 6.0 *on ubuntu 16.04*.